test/integration/scheduler/queue_test.go

```diff
@@ -49,127 +47,6 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-func TestServiceAffinityEnqueue(t *testing.T) {
```

Comment:
This test was added to test event registration.

@Huang-Wei is this covered somewhere else? ServiceAffinity is not supported in v1beta2 and above.

Reply:
yeah, `ServiceAffinity` was related to several packages, I was plan to remove it in another pr, this pr is also good. Listening to your guys for some advices.

Reply:
> @Huang-Wei is this covered somewhere else?

Nope, this is the only integration test to cover event registration (for core API resource).

We should replace it with another test case, like a volume event to trigger moving pods that failed the VolumeBinding plugin. Given it's irrelevant with this PR, just leave a TODO to me. SG?

```
// TODO(Huang-Wei): Add a test case to cover event registration for core API resources.
```

Reply:
Sounds good to me, but lets create an issue to track it and assign the todo to the issue.

Reply:
what should we do:
1. add a comment above
2. open a new pr to track the removal of ServiceAffinity
3. Add another test case as #105303 describes.

or we should combine `2` and `3` together in one pr?

Reply:
> or we should combine `2` and `3` together in one pr?

In one pr please.